src/components/SupportPage/SupportForm.jsx

```diff
@@ -0,0 +1,131 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Field, reduxForm } from 'redux-form';
+import { Button, Icon } from '@edx/paragon';
+
+import RenderField from '../RenderField';
+import StatusAlert from '../StatusAlert';
+
+import { isRequired, isValidEmail, maxLength512 } from '../../utils';
+
+class SupportForm extends React.Component {
+  renderErrorMessage() {
+    const { error: { message } } = this.props;
+
+    return (
+      <StatusAlert
+        className="mt-3"
+        alertType="danger"
+        iconClassName="fa fa-times-circle"
+        title="Unable to request more codes"
```

Comment:
Should this status alert message say something along the lines of, "Unable to send your request to the edX Customer Success team"?